src/drivers/gpio/bcm283x/bcm283x_gpio.h

```diff
@@ -0,0 +1,26 @@
+
+#ifndef SRC_DRIVERS_GPIO_BCM283X_BCM283X_GPIO_H_
+#define SRC_DRIVERS_GPIO_BCM283X_BCM283X_GPIO_H_
+
+#include <hal/reg.h>
+
+// Misc. GPIO pins of note
+//
+#define GPIO_ONBOARD_LED_PIZERO     47
+
+typedef enum _GpioFunc {
+    GFInput = 0,
+    GFOutput = 1,
+    GFAlt0 = 4,
+    GFAlt1 = 5,
+    GFAlt2 = 6,
+    GFAlt3 = 7,
+    GFAlt4 = 3,
+    GFAlt5 = 2
+} GpioFunc;
+
+#define GPU_IRQ     OPTION_GET(NUMBER,gpu_irq_int)
```

Comment:
You use OPTION_GET() but don't include its header  `#include  <framework/mod/options.h>`